src/app/services/config-options.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { LocalStorageService } from './local-storage.service';
+import { IConfigOptions } from '../models/config-options.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ConfigOptionsService {
```

Comment:
This is not exactly what was meant in the task:
`Создайте сервис ConfigOptionsService (core/services/config-options.service.ts), который должен хранить объект настроек (id, login, email, ...). Сервис должен предоставлять методы для установки и извлечения данных. Метод установки на вход принимает объект, а метод извлечения данных возвращает объект. Предусмотреть возможность установки подмножества свойств. Например, { id, login }.`

there should be configOptions object in the class, e.g. {id: 1, login: 'login', name: 'Vasya'}.
this.getConfigOptions() should return all config options;
this.getConfigOption(key)  is OK
this.setConfigOptions should receive object with partial update. E.g. it can be called
this.setConfigOptions({id: 2, name: 'Kolya'})